config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json

```diff
@@ -3,19 +3,23 @@
         "5.1":{
             "status": "supported",
             "latest-revision": {
-                "revision": "7",
-                "release-date": "12 July 2024",
-                "cves": "",
-                "golang": "1.21.12",
+                "revision": "8",
+                "release-date": "3 October 2024",
+                "cves": "CVE-2024-2398, CVE-2024-37370, CVE-2024-37371",
+                "golang": "1.22.7",
                 "ubi": "",
-                "new-features": [],
+                "new-features": ["Updated to k8s 1.30 client libraries",
+                "Update CSI Spec to 1.9.0",
+                "Fix security issues for CSI sidecar livenessprobe which now will run as non-root user/group in Node Server pod.",
+                "Updates the following sidecar images, - csi-provisioner:v5.0.2, - csi-resizer:v1.11.2, - csi-snapshotter:v8.0.1, - csi-attacher:v4.6.1, - livenessprobe:v2.13.1, - csi-node-driver-registrar:v2.11.1"
```

Comment:
`Updates the following sidecar images, - csi-provisioner:v5.0.2, - csi-resizer:v1.11.2, - csi-snapshotter:v8.0.1, - csi-attacher:v4.6.1, - livenessprobe:v2.13.1, - csi-node-driver-registrar:v2.11.1` ---->

`Updated the following sidecar images: csi-provisioner:v5.0.2, csi-resizer:v1.11.2, csi-snapshotter:v8.0.1, csi-attacher:v4.6.1, livenessprobe:v2.13.1, csi-node-driver-registrar:v2.11.1`